libs/ngxtension/click-outside/src/click-outside.ts

```diff
@@ -50,13 +48,11 @@ export class ClickOutside implements OnInit {
 	ngOnInit() {
 		this.documentClick$
 			.pipe(
-				takeUntil(this.destroy$),
```

Comment:
How does removing takeUntil remove a memory leak? 
I don't understand this tbh. 

We want to stop listening to documentClick on directive destroy, the injectDocumentClick is registered in root scope, so that won't ever be unsubscribed afaik.